tests/test_api_v1.py

```diff
@@ -1816,58 +1816,3 @@ def test_get_request_logs_not_configured(app, client, db, worker_auth_env):
     rv = client.get(f"/api/v1/requests/{request_id}")
     assert rv.status_code == 200
     assert "logs" not in rv.json
-
-
-@pytest.mark.parametrize(
```

Comment:
I don't see a value in retaining this. The only reason this was added was for the purpose of testing out the flag. Instead of editing at the time of flag removal, this should have been removed altogether. The scenarios being tested are covered in https://github.com/release-engineering/cachito/blob/master/tests/test_api_v1.py#L64 and removing this doesn't affect coverage either.

This is already being handled in a separate commit as part of this PR.

Reply:
I just re-read your comment. I still feel that we don't need a test for something that doesn't exist as part of the codebase/workflow anymore. If we were to follow that approach, we could end up with infinite unit tests.

Reply:
The flag still exists, it is just marked as inactive in the database and this test could be used to make sure the migration ran properly and the flag remains deactivated or that its usage introduce breakage.

> If we were to follow that approach, we could potentially end up with infinite unit tests.

Unit test completeness should not be seen as a bad thing.  Since the test is already there, we might as well use it instead of just dropping it. It may show itself to be useless. It may also prevent us introducing bugs in the future.

Reply:
I see your point, however the test can't be left alone since introducing a new task will require this to be modified as well and upon doing so it's a duplication of an existing test.

If we wanted to test the migration indirectly, as you mentioned it should have been done separately, probably when we disabled the flag.

According to the functionality, passing an inactive flag will fail the request on https://github.com/release-engineering/cachito/blob/master/cachito/web/models.py#L814 and it won't even reach the code we're testing here.

Hence, I'll remove it from this module. If we want to test the same functionality, we can add it back in the future as part of other modules while testing the flag existence.